scripts/word2md.js

```diff
@@ -47,6 +47,7 @@ function convertDocumentToMarkdown(doc) {
     var tableColumnCount;
     var tableCellIndex;
     var columnAlignment = [];
+    var imageCount = 0;

```

Comment:
I don't see this in the original `.ts`. Actually, why is this `.js` file even included here?